tasks/README.md

```diff
@@ -161,4 +162,24 @@ Add a webhook to your GitHub project on the Settings → Webhooks page of your p
  * Change the Content Type to `application/json`.
 
  * Select "Let me select individual events" and let the hook run on "Branch or
-   tag creation", and nothing else.
+   tag creation", "Pull requests", and "Statuses".
+
+See [GitHub development documentation](https://developer.github.com/webhooks/)
+for more information.
+
+### Automated releases
+
+Whenever a new release tag gets pushed to a project a creation event is received
```

Comment:
It's "create", no? Let's please match the official event names to avoid confusion.